Dockerfile.nethealth

```diff
@@ -17,16 +17,22 @@ ARG BUILD_VERSION
 ARG RIGGING_IMAGE
 
 FROM $BUILD_IMAGE as build
+RUN set -ex && apt-get update && apt-get install -y libcap2-bin && apt-get clean && rm -rf /var/lib/apt/lists/*
 ADD ./ /go/src/github.com/gravitational/satellite/
 RUN cd /go/src/github.com/gravitational/satellite/ && BUILD_VERSION=${BUILD_VERSION} go run mage.go build:nethealth
 
+
 #
 # Build nethealth container
 #
 FROM ${BASE_IMAGE}
-RUN apt-get update && apt-get install -y libcap2-bin && apt-get clean && rm -rf /var/lib/apt/lists/*
+
+COPY --from=build /sbin/getcap /sbin
+COPY --from=build /sbin/setcap /sbin
+COPY --from=build /lib/x86_64-linux-gnu/libcap.so.2 /lib
```

Comment:
That's copy actions are dangerous because base image and build image could be incompatible.

Reply:
Yea, unfortunately capability bits don't seem to be copiable (I suspect this is a design choice, so that a copied binary doesn't accidentally maintain capabilities that it shouldn't have). So I don't think this is the end of the world, especially going into a distroless container.

Reply:
So if its unsafe to copy libcap to our image, will nethealth be able to function the same without `cap_net_raw`?